webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
Is that what's intended here? Block until the navigation is complete but before the new document has started parsing?